pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "jupyter-schemas"
+dynamic = ["version"]
+description = 'Jupyter schemas'
+readme = "README.md"
+requires-python = ">=3.8"
+license = "MIT"
+keywords = []
+authors = [{ name = "Zach Sailer", email = "zsailer@apple.com" }]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+dependencies = ["jupyter_core"]
```

Comment:
It's important to note here that we are making `jupyter_core` a dependency of `jupyter_schemas`. This means we (ideally) won't have any schemas in jupyter_core, which would create a circular dependency.

I think this is fine, but I'm flagging here in case anyone else has concerns.